front/src/modules/timesStops/ReadOnlyTime.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import type { CellProps } from 'react-datasheet-grid/dist/types';
+
+import type { TimeExtraDays } from 'modules/trainschedule/components/ManageTrainSchedule/types';
+
+type ReadOnlyTimeProps = CellProps<TimeExtraDays | null | undefined, string>;
+
+const ReadOnlyTime = ({ rowData }: ReadOnlyTimeProps) => {
+  const { time, daySinceDeparture, dayDisplayed } = rowData || {};
+  if (time) {
+    const fullString =
+      daySinceDeparture !== undefined && dayDisplayed ? `${time} J+${daySinceDeparture}` : time;
```

Comment:
Perhaps this ‘J+’ should be included in the translation files. To get ‘D+’ in English

Reply:
done